hotelReservation/helm-chart/hotelreservation/templates/_baseDeploymentMongoDB.tpl

```diff
@@ -31,65 +31,73 @@ spec:
       {{- end }}
     spec:
       containers:
-      {{- with .Values.container }}
-      - name: "{{ .name }}"
-        image: {{ .dockerRegistry | default $.Values.global.dockerRegistry }}/{{ .image }}:{{ .imageVersion | default $.Values.global.defaultImageVersion }}
-        imagePullPolicy: {{ .imagePullPolicy | default $.Values.global.imagePullPolicy }}
+      - name: "{{ .Values.container.name }}"
+        image: "{{ .Values.container.image }}:{{ .Values.container.imageVersion }}"
+        imagePullPolicy: {{ .Values.container.imagePullPolicy | default $.Values.global.imagePullPolicy }}
```

Comment:
Is there a reason the `.dockerRegistry` value is no longer being used?  Also, the default value for `.imageVersion` has been dropped.